Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+
+
+const CardContainer = () => {
+return (
+    <div className='cardContainer' href='https://www.reactjs.org'>
```

Comment:
putting an `href` property on this div doesn't actually do anything. you still need an `<a>` tag to create a hyperlink to another page.